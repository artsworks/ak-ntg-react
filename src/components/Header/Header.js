import React from 'react';
import getSvg from '../../utils/getSvg';

function Header() {
    return (
        <header className="App__header">
            <div className="App__header-logo">
                {getSvg(99)}
            </div>
            <div className="App__header-racing">
                {getSvg(1)}
                <p>Racing</p>
            </div>
        </header>
    );
}

export default Header;