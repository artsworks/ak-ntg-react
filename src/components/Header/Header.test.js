import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('renders headers correctly', () => {
    const tree = renderer
        .create(<Header></Header>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});