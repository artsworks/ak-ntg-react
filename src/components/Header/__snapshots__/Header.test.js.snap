// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders headers correctly 1`] = `
<header
  className="App__header"
>
  <div
    className="App__header-logo"
  >
    <svg
      aria-hidden="true"
      className="cb-icon cb-icon--raw cb-icon--glyph-logo"
      height="32"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 274 158"
      width="32"
    >
      <defs>
        <linearGradient
          gradientTransform="matrix(1 0 0 1 -1.6539 0)"
          gradientUnits="userSpaceOnUse"
          id="logo-gradient"
          x1="151.5867"
          x2="270.7755"
          y1="123.8671"
          y2="4.6783"
        >
          <stop
            offset="0"
            stopColor="#23b4b7"
          />
          <stop
            offset=".8882"
            stopColor="#23b4b7"
            stopOpacity="0"
          />
        </linearGradient>
      </defs>
      <path
        d="M0 115.56V43.89h18.17c9.96 0 17.14 5.53 17.14 16.28v5.02c0 4.92-2.46 10.55-7.08 12.7 5.13 2.46 7.7 6.96 7.7 13.62v6.14c0 10.75-6.57 17.92-17.55 17.92H0zm12.32-41.67h3.7c4.52 0 6.98-3.17 6.98-7.37v-4.71c0-4.2-2.36-7.58-6.98-7.58h-3.7v19.66zm0 31.33h4.41c4.52 0 6.98-3.07 6.98-7.99V91.3c0-4.81-2.57-7.68-7.19-7.68h-4.21v21.6z"
      />
      <path
        d="M39.48 115.56V43.89H71.3v10.65H51.8v17.41h12.52V82.6H51.8v22.32h20.22v10.65H39.48z"
      />
      <path
        d="M98.28 115.56H86.06V54.53H74.97V43.89h34.39v10.65H98.28v61.02z"
      />
      <path
        d="M194.8 158c-43.67 0-79.2-35.44-79.2-79s35.53-79 79.2-79S274 35.44 274 79s-35.53 79-79.2 79zm0-145.33c-36.67 0-66.5 29.76-66.5 66.33s29.83 66.33 66.5 66.33 66.5-29.76 66.5-66.33-29.83-66.33-66.5-66.33z"
        fill="url(#logo-gradient)"
      />
      <path
        d="M144.76 109.09V48.91h20.81v6.96h-12.81v17.2h8.51v6.88h-8.51v22.18h13.16v6.96h-21.16z"
      />
      <path
        d="M193.51 109.09h-8.08l-1.2-11.52h-6.88l-1.29 11.52h-7.48l7.31-60.19h10.4l7.22 60.19zm-12.55-43.16l-2.92 25.54h5.59l-2.67-25.54z"
      />
      <path
        d="M217.5 69.03h-7.4v-9.54c0-2.92-1.12-4.38-3.44-4.38-2.06 0-3.27 1.63-3.27 3.96v5.42c0 3.61 1.2 6.53 3.53 8.77l5.33 5.07c3.96 3.7 5.93 8.68 5.93 15.13V98c0 6.96-4.39 11.69-11.52 11.69-7.14 0-11.35-4.39-11.35-11.26V86.82h7.48v11.61c0 2.67 1.29 4.39 3.7 4.39 2.32 0 3.61-1.63 3.61-4.13v-5.5c0-3.96-1.2-6.96-3.53-9.11l-4.99-4.64c-4.21-3.87-6.28-8.86-6.28-14.87V59.5c0-6.88 4.47-11.18 11.26-11.18 7.14 0 10.92 4.3 10.92 10.83v9.88z"
      />
      <path
        d="M235.72 109.09h-8V84.42l-9.11-35.51h8.68l4.47 22.87 4.9-22.87h8.17l-9.11 35.25v24.93z"
      />
    </svg>
  </div>
  <div
    className="App__header-racing"
  >
    <svg
      aria-hidden="true"
      height="32"
      preserveAspectRatio="xMidYMid meet"
      viewBox="0 0 32 32"
      width="32"
    >
      <g>
        <path
          d="M17.72,28.24a.54.54,0,0,1-.48-.32c-.08-.24-.24-.56-.32-.8-.4-.88-.88-1.76-1.28-2.65-.24-.4-.4-.72-.64-1.12l-.24-.4-.48-1a7.06,7.06,0,0,1-.64-2.65.55.55,0,0,1,.56-.56.6.6,0,0,1,.56.56,5.09,5.09,0,0,0,.56,2.17c.16.32.24.56.4.88l.24.4c.16.4.4.72.56,1.12.48.88,1,1.76,1.44,2.73a9.41,9.41,0,0,1,.4.88c.08.32,0,.64-.32.72Zm6.58-5.86a1.53,1.53,0,0,1-1-.56,4.41,4.41,0,0,1-.56-.72c-.08-.16-.24-.24-.32-.4a11.41,11.41,0,0,1-3.13-1.12l-1.12-.16c-.56,0-1.2-.08-1.85-.08a1.51,1.51,0,0,1-.64-.16c-2-.56-2.81-2-2.81-4.25v-.56a.52.52,0,0,1,1,.08v.48c0,1.76.56,2.65,2,3.13a1.07,1.07,0,0,0,.4.08c.56,0,1.2.08,1.85.08l1,.08a.76.76,0,0,1,.56.16,8.18,8.18,0,0,0,2.89,1,1,1,0,0,1,.56.4,1.75,1.75,0,0,0,.32.4c.16.24.32.4.48.64l.16.16.16-.16A1.61,1.61,0,0,1,26,20.54a1.89,1.89,0,0,0,.16-.64,6.07,6.07,0,0,0-.08-1.2v-.24l-.4-.4a4.2,4.2,0,0,1-.8-.88,22.67,22.67,0,0,0-2.65-3,2.28,2.28,0,0,1-.88-1.68l-.08-.16a2.5,2.5,0,0,0-.56-.72c-.32-.4-.56-.64-.72-.88s-.48-.56-.8-1a.6.6,0,0,1,0-.72,6.82,6.82,0,0,0,1.12-3c.08-.24.08-.48.16-.64a.53.53,0,0,1,.64-.4.57.57,0,0,1,.4.72,5.11,5.11,0,0,0-.16.56,9.16,9.16,0,0,1-1,3,4.63,4.63,0,0,1,.64.8c.24.24.4.56.72.88a8.4,8.4,0,0,1,.64.88,1.51,1.51,0,0,1,.32.72,1,1,0,0,0,.48.8,25.35,25.35,0,0,1,2.81,3.13,6.13,6.13,0,0,0,.72.72l.4.4a1,1,0,0,1,.32.56v.32c0,.48.08.88.08,1.44a3.39,3.39,0,0,1-.24,1v.4a.82.82,0,0,1-.4.48.89.89,0,0,1-.64,0,.61.61,0,0,0-.8.08A2.46,2.46,0,0,1,24.29,22.39ZM5,21.1H4.8a.53.53,0,0,1-.4-.64l.16-.56A19.17,19.17,0,0,1,8,12.36a14,14,0,0,1,5-4.17,7.15,7.15,0,0,1,3.13-.72,6,6,0,0,1,1.2-1.52A4.45,4.45,0,0,1,18,5.46l.16-.08A1.88,1.88,0,0,0,19,4.5a.76.76,0,0,1,.8-.32.93.93,0,0,1,.32.8,8.25,8.25,0,0,1-1.2,3.77A.57.57,0,0,1,18,8a4.4,4.4,0,0,0,.64-1.6l-.08.08c-.16.16-.32.24-.48.4a2.86,2.86,0,0,0-1,1.28.51.51,0,0,1-.48.4h-.48a5.94,5.94,0,0,0-2.57.56A14.92,14.92,0,0,0,8.89,13a17.27,17.27,0,0,0-3.21,7.06l-.16.56A.59.59,0,0,1,5,21.1Z"
        />
      </g>
    </svg>
    <p>
      Racing
    </p>
  </div>
</header>
`;
